README.md

```diff
@@ -18,10 +18,33 @@ In this Quick Start, we will provide individual examples for each Synopsys secur
 
 These workflows will:
 - Validate Scanning platform-related parameters like project and stream
-- Download the Synopsys Bridge and related adapters
 - Run corresponding Synopsys Bridge commands using the specified parameters
 - Synopsys solution functionality is invoked directly by the Synopsys Bridge, and indirectly by the Synopsys Action
 
+# Prerequisites
+
+Before configuring Synopsys Action into your workflow, note the following prerequisites:
+
+**GitHub Runner Setup:**
+
+- Runners are the machines that execute jobs in a GitHub Actions workflow. To use GitHub runners in your project, GitHub Actions must be enabled for a repository/organization settings in order for required workflows to run (Repository Settings → SelectActions → General → Actions permissions)
+- GitHub runner can be Self-hosted or GitHub-hosted. For installing Self-hosted runners refer [Self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners) and for installing GitHub-hosted runners refer [GitHub-hosted Runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners)
```

Comment:
We cannot install github-hosted runner. So may be we can rephrase "installing Github-hosted runners" to "using Github hosted runner"

Reply:
https://docs.github.com/en/enterprise-cloud@latest/actions/hosting-your-own-runners/managing-self-hosted-runners/adding-self-hosted-runners#adding-a-self-hosted-runner-to-an-organization
GitHub is referring it as installation only